lib/permits/resolvers.ts

```diff
@@ -0,0 +1,53 @@
+import { ApolloError } from 'apollo-server-errors'; // Apollo error
+import { Resolver } from '@lib/resolvers'; // Resolver type
+import { PermitAlreadyExistsError } from '@lib/permits/errors'; // Employee errors
+import { DBErrorCode } from '@lib/db/errors'; // Database errors
+
+/**
+ * Query all the permits in the internal-facing app
+ * @returns All permits
+ */
+export const permits: Resolver = async (_parent, _args, { prisma }) => {
+  const permits = await prisma.permit.findMany();
+  return permits;
+};
+
+/**
+ * Create an RCD permit
+ * @returns Status of operation (ok, error)
+ */
+export const createPermit: Resolver = async (_, args, { prisma }) => {
+  const {
+    input: { rcdPermitId, expiryDate, receiptId, active, applicationId, applicantId },
+  } = args;
+
+  let permit;
+  try {
+    permit = await prisma.permit.create({
+      data: {
+        rcdPermitId,
+        expiryDate,
+        receiptId,
+        active,
+        applicationId,
+        applicantId,
+      },
```

Comment:
if you pull out input above, you can unpack it here instead to save a few lines:
```suggestion
      data: { ...input },
```